lms/djangoapps/instructor/permissions.py

```diff
@@ -41,7 +41,7 @@
 perms[GENERATE_BULK_CERTIFICATE_EXCEPTIONS] = is_staff
 perms[GIVE_STUDENT_EXTENSION] = HasAccessRule('staff')
 perms[VIEW_ISSUED_CERTIFICATES] = HasAccessRule('staff') | HasRolesRule('data_researcher')
-perms[CAN_RESEARCH] = HasRolesRule('data_researcher')
+perms[CAN_RESEARCH] = is_staff | HasRolesRule('data_researcher')
```

Comment:
Why `is_staff` and not `HasAccessRule('staff')` like the other `perms` here, eg: `perms[ENROLLMENT_REPORT]`?

Reply:
@stvstnfrd I believe `HasAccessRule('staff')` also includes the `staff` course role, as well as the global staff. In this case, only the global staff or the people with the `data_researcher` role should be able to access the tab.

Reply:
Makes sense. We _may_ want to include a comment to make it clear why this line is subtly different than the others, but LGTM.